src/payment/components/ProductLineItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ProductLineItem extends React.PureComponent {
+  renderSeatType(seatType) {
+    const selectedSeatType = (() => {
+      switch (seatType) {
+        case 'professional':
+        case 'no-id-professional':
+          return 'Professional Certificate';
```

Comment:
Let's localize this using `<FormattedMessage ... />` from `@edx/frontend-i18n`

Reply:
Noted, updated `case 'verified-certificate':
          return (<FormattedMessage
            id="payment.productlineitem.verified.certificate"
            defaultMessage="Verified Certificate"
            description="Course certificate type on product details section"
          />);`